language-design.md

```diff
@@ -28,7 +75,69 @@ you know a poorly designed language? What are the symptoms?
 
 **Response**
 
-
+A well-designed language is made up of many things. A few symptoms are 
+intuitiveness, ability for growth, well-defined semantics and langauge building 
+blocks. There are a few more symptoms when talking about a well-designed 
+language for industry-use and some other symptoms if we are talking about a DSL.
+
+We defined intuitiveness to mean something that is relatively easy to read and 
+write. By "easy to read and write" we don't mean easy to read and write for 
+someone who's never programmed before (which seems to be what Pavlus is 
+advocating in his article [Pavlus, 2012]). Rather, for someone's who's seen or 
+used a few languages, maybe taken CS5 and CS60, can they read and possibly 
+start writing code in this language quickly?
+
+A language that is well-designed does not always have to be extensible and 
+allow for growth, but allowing the language to be modified by people other than 
+the creators allows for a bigger community which makes the language more 
```

Comment:
Modern Javascript is quite nice (and jQuery is often not necessary anymore for many modern frameworks)